js/grumbles/show.controller.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+(function(){
+  angular
+  .module("grumbles")
+  .controller("GrumbleShowController", [
+    "$stateParams",
+    GrumbleShowControllerFunction
+  ]);
+
+  function GrumbleShowControllerFunction($stateParams){
+    this.grumble = grumbles[$stateParams.id];
+  }
+
+}());
```

Comment:
code looks good! I don't understand your comment in the PR "My U..." Let me know if you want to pair on this after break